napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -354,6 +442,41 @@ def open_ndisplay_camera_popup(self) -> None:
         # Reposition popup, must be done after all widgets are added
         self._position_popup_inside_viewer(popup, self.ndisplayButton)
 
+    def _update_orientation(
+        self,
+        orientation_enum: (
+            DepthAxisOrientation
+            | VerticalAxisOrientation
+            | HorizontalAxisOrientation
+        ),
+        value: (
+            DepthAxisOrientationStr
+            | VerticalAxisOrientationStr
+            | HorizontalAxisOrientationStr
+        ),
```

Comment:
Note: I found the previous auto-formatting funky, because there was no indent on the continuation lines:

```python
         orientation_enum: DepthAxisOrientation
         | VerticalAxisOrientation
         | HorizontalAxisOrientation,
         value: DepthAxisOrientationStr
         | VerticalAxisOrientationStr
         | HorizontalAxisOrientationStr,
```

I added parens as a "guide" for the formatter (ruff and black allow this for certain expressions), and it worked, so I pushed the changes.